distributed/tests/test_client.py

```diff
@@ -3354,6 +3354,12 @@ def test_get_processing(c, s, a, b):
     x = yield c.scheduler.processing(workers=[a.address])
     assert isinstance(x[a.address], list)
```

Comment:
We can probably remove this code above now that `client.processing` works directly.

Reply:
Done. I glanced through the other tests in `test_client.py`. A lot of them access methods similar to `processing` like `has_what` through the `scheduler` attribute. I'm not sure if that's by design or if that's how it was done in an earlier version of the API.